spec/features/settings/ext_cert_feature_spec.rb

```diff
@@ -0,0 +1,233 @@
+require "rails_helper"
+
+# rubocop:disable RSpec/ExampleLength
+describe "Feature: External Cerificate settings", js: true do
+
+  let!(:user) { create(:user) }
+  let!(:fixture_path) { RSpec.configuration.fixture_path }
+
+  let(:ssl_cert_file_a) { File.join(fixture_path, "ext_cert_ssl_a.pem") }
+  let(:ssl_key_file_a) { File.join(fixture_path, "ext_cert_key_a.pem") }
+  let(:ssl_cert_file_b) { File.join(fixture_path, "ext_cert_ssl_b.pem") }
+  let(:ssl_key_file_b) { File.join(fixture_path, "ext_cert_key_b.pem") }
+  let(:ssl_cert_file_malformed) { File.join(fixture_path, "ext_cert_ssl_mal.pem") }
+  let(:ssl_key_file_malformed) { File.join(fixture_path, "ext_cert_key_mal.pem") }
+  let(:expired_cert) { File.join(fixture_path, "expired_cert.pem") }
+  let(:key_for_expired_cert) { File.join(fixture_path, "key_for_expired_cert.pem") }
+
+  before do
+    setup_done
+    login_as user, scope: :user
+  end
+
+  describe "#index" do
+    before do
+      visit settings_ext_cert_index_path
+    end
+
+    # Success Conditions
+
+    it "barely does the bare minimum" do
```

Comment:
'bare' essentially used twice here; 'does the bare minimum' would be sufficient

Reply:
Done